src/chains/admin.py

```diff
@@ -38,6 +41,18 @@ class ChainAdmin(admin.ModelAdmin[Chain]):
     )
     inlines = [FeatureInline, GasPriceInline, WalletInline]
 
+    def get_readonly_fields(
+        self, request: HttpRequest, obj: Optional[Chain] = None
+    ) -> List[str]:
+        if (
+            request.user.has_perm("chains.change_only_warning")
+            and not request.user.is_superuser
```

Comment:
Why don't we allow a superuser to change this field? (even without the permission)

Reply:
(Please refer to the [previous comment](https://github.com/safe-global/safe-config-service/pull/688#discussion_r1018095602) for reference)

This function (`get_readonly_fields`) returns the read-only fields, so for an user which have the `change_only_warning` and is not superadmin, will return all the `Chain` fields except `warning`. For superusers, it will return `[]`, so all the fields are editable. In short, we allow superusers to change this field.

(I know this code is not super intuitive to read, please let me know if you find a better approach).